hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Listing.java

```diff
@@ -763,6 +766,61 @@ public boolean accept(FileStatus status) {
     }
   }
 
+  /**
+   * Accept all entries except the base path and those which map to S3N
+   * pseudo directory markers and CSE instruction file.
+   */
+  public static class AcceptFilesOnlyExceptCSEInstructionFile implements FileStatusAcceptor {
```

Comment:
* cut the S3N support in this one, and make the class final
* move to CSEV1CompatibleS3AFileSystemHandler class

Reply:
ack

Reply:
I am removing instruction file skipping logic